src/constants.py

```diff
@@ -27,4 +27,3 @@
 SYNAPSE_PORT = 8008
 SYNAPSE_SERVICE_NAME = "synapse"
```

Comment:
`SYNAPSE_SERVICE_NAME` seems to be used only in pebble.py (and in the tests). Can you move it there?

Reply:
Good point, I'll refactor all of the constants like this :
- If it's only used in 1 file, I'll put it directly in that file
- since most of them are used in `workload` and `api` as you mentioned, and there's a dependency between them, if a constant is used in more than 2 places, I'll put it in `api` (I think it's better to put all imported stuffs in one place in case we need to change them later)

Then I'll remove this file and do the same for the test constants. What do you think? @arturo-seijas  / @amandahla

Reply:
Sounds good to me

Reply:
moved those constants and deleted the two constant files